java/pypowsybl/src/main/java/com/powsybl/python/rao/RaoCFunctions.java

```diff
@@ -127,8 +125,11 @@ public static PyPowsyblApiHeader.ArrayPointer<CCharPointer> serializeRaoResultsT
             RaoResult raoResult = ObjectHandles.getGlobal().get(raoResultHandle);
             Crac crac = ObjectHandles.getGlobal().get(cracHandle);
             try (ByteArrayOutputStream output = new ByteArrayOutputStream()) {
-                new RaoResultJsonExporter().exportData(
-                    raoResult, crac, Set.of(Unit.MEGAWATT), output);
+                ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+                Properties properties = new Properties();
+                properties.setProperty("rao-result.export.json.flows-in-amperes", "true");
```

Comment:
Do we really want to hardcode those two properties ?

Reply:
Cleaned

Reply:
Discussed with @phiedw. We let it as it is for this version and maybe later on add a parameter.